CTestJenkins.cmake

```diff
@@ -46,7 +46,8 @@ if(${CMAKE_MAJOR_VERSION} GREATER 3 OR (${CMAKE_MAJOR_VERSION} EQUAL 3 AND ${CMA
     set(QUIET_ QUIET)
 endif()
 
-ctest_empty_binary_directory(${CTEST_BINARY_DIRECTORY})
+# Remove 
+# ctest_empty_binary_directory(${CTEST_BINARY_DIRECTORY})
```

Comment:
This comment was for testing. Remove it.

Reply:
Removed